.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -36,109 +35,109 @@ jobs:
+              kubectl get pod -n kube-system -l app=privileged-daemonset,os=windows -owide
+              pods=`kubectl get pod -n kube-system -l app=privileged-daemonset,os=windows --no-headers | awk '{print $1}'`
+              for pod in $pods; do
+                kubectl exec -i -n kube-system $pod -- powershell "Restart-Service kubeproxy"
+                kubectl exec -i -n kube-system $pod -- powershell "Get-Service kubeproxy"
+              done
+            name: kubeproxy
+            displayName: Restart Kubeproxy on Windows nodes
+            retryCountOnTaskFailure: 3
       - ${{ if eq(parameters.datapath, true) }}:
-        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
-          parameters:
-            testName: Datapath
-            name: datapath
-            ginkgoFocus: '(.*).Networking.should|(.*).Networking.Granular|(.*)kubernetes.api'
-            ginkgoSkip: 'SCTP|Disruptive|Slow|hostNetwork|kube-proxy|IPv6'
-            os: ${{ parameters.os }}
-            processes: 8
-            attempts: 10
+          - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+            parameters:
+              testName: Datapath
+              name: datapath
+              ginkgoFocus: "(.*).Networking.should|(.*).Networking.Granular|(.*)kubernetes.api"
+              ginkgoSkip: "SCTP|Disruptive|Slow|hostNetwork|kube-proxy|IPv6"
+              os: ${{ parameters.os }}
+              processes: 8
+              attempts: 10
       - ${{ if eq(parameters.dns, true) }}:
-        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
-          parameters:
-            testName: DNS
-            name: dns
-            ginkgoFocus: '\[sig-network\].DNS.should'
-            ginkgoSkip: 'resolv|256 search'
-            os: ${{ parameters.os }}
-            processes: 8
-            attempts: 3
+          - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+            parameters:
+              testName: DNS
+              name: dns
+              ginkgoFocus: '\[sig-network\].DNS.should'
+              ginkgoSkip: "resolv|256 search"
+              os: ${{ parameters.os }}
+              processes: 8
+              attempts: 3
       - ${{ if eq(parameters.portforward, true) }}:
-        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
-          parameters:
-            testName: Kubectl Portforward
-            name: portforward
-            ginkgoFocus: '\[sig-cli\].Kubectl.Port'
-            ginkgoSkip: ''
-            os: ${{ parameters.os }}
-            processes: 8
-            attempts: 3
+          - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+            parameters:
+              testName: Kubectl Portforward
+              name: portforward
+              ginkgoFocus: '\[sig-cli\].Kubectl.Port'
+              ginkgoSkip: "port-forward should keep working after detect broken connection" # affecting k8s 1.32 https://github.com/kubernetes/kubernetes/issues/129803. Note: retry this test in 1.33 with timeout extended
```

Comment:
good to have a task in ADO tracking this so that we don;t miss it. (if not already have one)